script/import_scripts/bbpress.rb

```diff
@@ -84,7 +84,7 @@ def import_users
           id: u["id"].to_i,
           username: u["user_nicename"],
           email: u["user_email"].downcase,
-          name: u["display_name"],
+          name: u["display_name"].length > 0 ? u["display_name"] : u['user_nicename'],
```

Comment:
rails has a `.presence` helper that return the string only if it has some content

```rails
name: u["display_name"].presence || u["user_nicename"]
```